client/src/App.tsx

```diff
@@ -1,17 +1,23 @@
 import "./App.css";
+import React, { useState } from "react";
 import AppRouter from "./routers/Router";
 import Navbar from "./components/navbar";
+import ThemeContext, { Theme } from "./context/ThemeContext";
 
 function App() {
+  const [theme, setTheme] = useState("light");
```

Comment:
Would it be better to use a enum for the Theme, like Theme.LIGHT or Theme.DARK

Reply:
Makes sense